specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/examples/ApiManagementCreateWorkspacePremiumGatewayKubernetesManaged.json

```diff
@@ -0,0 +1,105 @@
+{
+  "parameters": {
+    "gatewayName": "apimGateway1",
+    "resourceGroupName": "rg1",
+    "api-version": "2024-10-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "parameters": {
+      "properties": {
+        "backend": {
+          "subnet": {
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.Network/virtualNetworks/vn1/subnets/sn1"
+          }
+        },
+        "virtualNetworkType": "External"
```

Comment:
We don't support external VNET so let's remove this.

Reply:
Done.

Reply:
I still see it though

Reply:
```suggestion
```